projects/workflows-creator/src/lib/builder/group/group.component.ts

```diff
@@ -614,6 +626,26 @@ export class GroupComponent<E> implements OnInit, AfterViewInit {
       callback(response);
     }
   }
+
+  updateSecondVariable(
+    event: any,
+    inputType: InputTypes,
+    dateTimeField?: string,
+  ) {
+    //if inputType->dateTime
+    switch (inputType) {
+      case InputTypes.DateTime:
+        if (dateTimeField == 'date') {
+          this.dateTime.date = event;
+        } else {
+          this.dateTime.time = event;
+        }
```

Comment:
ternary operator here

Reply:
We are updating different entity in 'if' and in 'else', so I suppose it is better like this only